### ISSUE TITLE
Original quote with ellipsis in it is not getting highlighted in the scripture card

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/gateway-edit#177]
Two issues here:
1. The ellipsis is not displayed in the tN card for the quote instead it shows "&" character . Eg, tit 1:6, 1:15, 1:16

![Screen Shot 2021-04-28 at 12.04.20 AM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/275b5c31-1a30-4e79-87cc-04ef361a584a)

2. The quote is not highlighted in the scripture card.

**Comment by birchamp** [comment]
@RobH123 has the ellipsis been replace by the ampersand?

**Comment by RobH123** [comment]
@birchamp Yes, in the new TSVs, it uses ' & ' (space, ampersand, space) for indicating non-contiguous quote portions. See https://forum.door43.org/t/parascriptural-tab-separated-value-format-specification-v2/870.
